web/api/v1/api_test.go

```diff
@@ -1237,6 +1330,66 @@ func testEndpoints(t *testing.T, api *API, tr *testTargetRetriever, es storage.E
 			},
 			responseAsJSON: `{"resultType":"vector","result":[]}`,
 		},
+		// limit disabled
```

Comment:
The remaining tests below (through line 1392) don't make sense as they only have one metric. If you repeat the tests above with the different permutations as recommended, you can remove all the tests below.